test/src/test/java/org/corfudb/integration/cluster/Harness/Node.java

```diff
@@ -30,6 +30,13 @@ public Node(String address, String clusterAddress, String logPath) {
         this.start = new StartAction(this);
     }
 
+    public Node(String address, String logPath) {
```

Comment:
There is a 3 param constructor, we can call this to reduce code rep..

Reply:
Done.